ansible/group_vars/all

```diff
@@ -1058,6 +1058,59 @@ rundeck:
     ttl: 60
     overwrite: yes
 
+# Spark Prototype configuration
+spark_proto:
+  skip: true
+  elastic_load_balancer:
+    name: "{{ vpc_short_name }}-sparkpto-elb"
+    connection_draining_timeout: 600
```

Comment:
Just a design choice related to closing connections on deregistering instances. We have a fairly generous connection draining on all components that are likely to have long running connections (e.g.: reporting). 600 is a little bit extreme though and should be lowered.

http://docs.aws.amazon.com/ElasticLoadBalancing/latest/DeveloperGuide/config-conn-drain.html